darts/models/forecasting/ensemble_model.py

```diff
@@ -197,5 +197,21 @@ def ensemble(
     def min_train_series_length(self) -> int:
         return max(model.min_train_series_length for model in self.models)
 
+    @property
+    def extreme_lags(self):
+        extreme_lags_length = len(super().extreme_lags)
+
+        return [self._find_max_lag_or_none(i) for i in range(extreme_lags_length)]
+
+    def _find_max_lag_or_none(self, lag_id):
+        max_lag = None
+        for model in self.models:
+            curr_lag = model.extreme_lags[lag_id]
+            if max_lag is None or (
```

Comment:
I'm not sure if this respects the actual min/max of all extreme lags.

For example lags_future_covariates for one regression models could be [-3, -2], and for a second one [-1, 1]. 
The min lag for both models should be -3, and max lag +1. 
However, the current implementation would give min/max lags of -3 and -2

Reply:
Okay, I assumed that for a given lag all values must either be positive or negative, and abs() would be working then. I checked out the docs and I think that the new implementation fixes the problem